source/blog/2018-10-05-the-ember-times-issue-67.md

```diff
@@ -30,8 +30,19 @@ And don't forget to register on the [Hacktoberfest site](https://hacktoberfest.d
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Come Play in the Ember CodeSandbox 🏖️🏰](https://medium.com/@mikenorth/ember-community-meet-codesandbox-10a43076b3fa)
+
+[CodeSandbox](https://codesandbox.io) is an online code editor for the Web, and now for Ember as well! [Mike North](https://github.com/mike-north) has done a lot of work to get Ember CLI working inside of CodeSandbox so that we can develop, showcase, share live apps on the web! 
+
+What allowed this to happen is that CodeSandbox released server-side evaluated sandboxes which now allows CLI Node apps to run and be served by the site. 
+
+Of course we want to give a shoutout to [Guarav](https://github.com/Gaurav0) and the rest of the Ember Twiddle team for their amazing work. However the way these Twiddles work is by [emulating](https://github.com/ember-cli/ember-twiddle/blob/ac116eb20d1e6c2152313f865159c443a9e2bd6f/app/services/ember-cli.js#L261-L295) a node environment and not actually creating one. With CodeSandbox we get real Ember CLI out of the box.
```

Comment:
My name is spelled Gaurav

Reply:
Thanks @gaurav0 Sorry for misspelling your name :) I'll add a section on the update for Ember Twiddle as well.

Reply:
Fixed. We typically go with the GitHub handle per https://github.com/emberjs/website/blob/master/source/blog/CONTRIBUTING.md#writing-style-guide.